kikuchipy/__init__.py

```diff
@@ -20,26 +20,41 @@
 from kikuchipy.release import version as __version__
 
 
-# Attempt import only once
+# Attempt (and fail) import of optional dependencies only once
 try:
     import pyvista
 
     _pyvista_installed = True
 except ImportError:  # pragma: no cover
     _pyvista_installed = False
+
 try:
     import nlopt
 
     _nlopt_installed = True
 except ImportError:  # pragma: no cover
     _nlopt_installed = False
+
 try:
     from pyebsdindex import pcopt, ebsd_index
 
     _pyebsdindex_installed = True
 except ImportError:  # pragma: no cover
     _pyebsdindex_installed = False
 
+# PyOpenCL context available for use with PyEBSDIndex?
+try:
+    from pyebsdindex.opencl.openclparam import OpenClParam
+
+    clparams = OpenClParam()
+    clparams.get_context()
```

Comment:
I would use `pyopencl` directly instead, because the return of `clparams.get_context()` could be regarded as an implementation details of `pyebsdindex`.
I would expect that `pyopencl` have a clean approach to do this.

Reply:
Agreed. I've done this instead.